rpp30-three-musketeers/FEC
# Outfit testing

Refactored RelatedProducts / Outfit testing suite to separate concerns. 
- Add script to run tests for outfit / related products components in isolation
- Test rendering of static features
- Create placeholder file to test server response
    - TODO: use supertest to mock server responses
- installed adapter package to run Enzyme with React v.17
   - See https://www.npmjs.com/package/@wojtekmaj/enzyme-adapter-react-17 for installation and setup instructions
